qa/tasks/ceph_manager.py

```diff
@@ -945,29 +946,37 @@ def test_pool_min_size(self):
                          format(rand=rand_val))
                 if len(self.live_osds) > minup+1 and rand_val < 0.5:
                     # chose to knock out as many OSDs as we can w/out downing PGs
-                    
-                    most_killable = min(len(self.live_osds) - minup, m)
+
+                    most_killable = min(len(self.live_osds) - (minup + 1), m)
```

Comment:
Is minup simply being set incorrectly?  From the name, I would expect it to be ok to kill osds until minup are up.

Reply:
minup is set correctly already, the +1 is an overkill so changing this back to normal